.ci/cico_common.sh

```diff
@@ -0,0 +1,232 @@
+    yum -y install rh-maven33
+    yum install -y yum-utils device-mapper-persistent-data lvm2
+    yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
+    curl -sL https://rpm.nodesource.com/setup_10.x | bash -
+    yum-config-manager --add-repo https://dl.yarnpkg.com/rpm/yarn.repo
+    yum install -y docker-ce nodejs yarn gcc-c++ make
+    service docker start
+}
+
+mvn_build() {
+    set -x
+    if [[ $DO_NOT_IGNORE_TESTS == "true" ]]; then
+        scl enable rh-maven33 'mvn clean install -U -Pintegration -Dmaven.test.failure.ignore=false'
+    else
+        scl enable rh-maven33 'mvn clean install -U -Pintegration'
+    fi
+    if [[ $? -eq 0 ]]; then
+        echo 'Build Success!'
+    else
+        die_with  'Build Failed!'
+    fi
+}
+
+mvn_deploy() {
+    set -x
+    echo 'Going to deploy artifacts'
+    scl enable rh-maven33 "mvn clean deploy -DcreateChecksum=true  -Dgpg.passphrase=$CHE_OSS_SONATYPE_PASSPHRASE"
+    if [[ $? -eq 0 ]]; then
+        echo 'Deploy Success!'
+    else
+        die_with  'Deploy Failed!'
+    fi
+}
+
+
+
+gitHttps2ssh(){
+    #git remote set-url origin git@github.com:$(git remote get-url origin | sed 's/https:\/\/github.com\///' | sed 's/git@github.com://')
+    #git version 1.8.3 not support get-url sub-command so hardcode url
+    git remote set-url origin git@github.com:eclipse/che
+}
+
+
+setup_gitconfig() {
+  git config --global user.name "Vitalii Parfonov"
+  git config --global user.email vparfono@redhat.com
+}
+
+ # KEEP RIGHT ORDER!!!
+DOCKER_FILES_LOCATIONS=(
+    dockerfiles/endpoint-watcher
+    dockerfiles/keycloak
+    dockerfiles/postgres
+    dockerfiles/dev
+    dockerfiles/che
+    dockerfiles/dashboard-dev
+    dockerfiles/e2e
+)
+
+IMAGES_LIST=(
+    quay.io/eclipse/che-endpoint-watcher
+    quay.io/eclipse/che-keycloak
+    quay.io/eclipse/che-postgres
+    quay.io/eclipse/che-dev
+    quay.io/eclipse/che-server
+    quay.io/eclipse/che-dashboard-dev
+    quay.io/eclipse/che-e2e
+)
+
+REGISTRY="quay.io"
+ORGANIZATION="eclipse"
+
+buildImages() {
+    echo "Going to build docker images"
+    set -e
+    set -o pipefail
+    TAG=$1
+  
+    # stop / rm all containers
+    if [[ $(docker ps -aq) != "" ]];then
+        docker rm -f $(docker ps -aq)
+    fi
+
+    # BUILD IMAGES
+    for image_dir in ${DOCKER_FILES_LOCATIONS[@]}
+     do
+         bash $(pwd)/${image_dir}/build.sh --tag:${TAG} 
+         if [[ ${image_dir} == "dockerfiles/che" ]]; then
+           #CENTOS SINGLE USER
+           BUILD_ASSEMBLY_DIR=$(echo assembly/assembly-main/target/eclipse-che-*/eclipse-che-*/)
+           LOCAL_ASSEMBLY_DIR="${image_dir}/eclipse-che"
+           if [[ -d "${LOCAL_ASSEMBLY_DIR}" ]]; then
+               rm -r "${LOCAL_ASSEMBLY_DIR}"
+           fi
+           cp -r "${BUILD_ASSEMBLY_DIR}" "${LOCAL_ASSEMBLY_DIR}"
+           docker build -t ${REGISTRY}/${ORGANIZATION}/che-server:${TAG}-centos -f $(pwd)/${image_dir}/Dockerfile.centos $(pwd)/${image_dir}/
```

Comment:
Seems these scripts need to be updated since dashboard should not build che-server image.

Reply:
done